packaging_automation/tests/test_citus_package.py

```diff
@@ -28,6 +28,10 @@
     "el/8": 3,
     "ol/7": 2,
     "ol/8": 3,
+    "almalinux/9": 3,
+    "rockylinux/9": 3,
+    "elnew/9": 3,
+    "olnew/9": 3,
```

Comment:
why do we have those "new" suffixes ?

Reply:
it was my fault. Removed the new suffixes.